client-netty/README.md

```diff
@@ -1,62 +1,156 @@
 ## WebPush Client based on Netty
+This module contains both a WebPush client and a command line applications called the WebPush Console.
 
+The WebPush Console is intended to be used during development and manual testing of a WebPush server. Currently, it works
+with the [node-webpush-server](https://github.com/kitcambridge/node-webpush-server) and the AeroGear
+[webpush-server](../server-netty).
 
 ## Using the WebPush console
-A very basic console/command line application is available to assist in development and testing of the server. 
+A very basic console/command line application is available to assist in development and testing of the server.
 
-### Start the server
+__NOTE__:
+The [node-webpush-server](https://github.com/kitcambridge/node-webpush-server) uses the Transport Layer Security (TLS)
+extension Next Protocol Negotiation (NPN) and does currently not support Application-Layer Protocol Negotiation (ALPN).
+There is no NPN support for Java 1.8 so it is required that Java 1.7 be used to start the WebPush Console. This cannot
+be done as a choice at runtime as the jar files for either NPN or ALPN must be added to the Java boot classpath.
+
+### Start the AerogGear server
+__Java 1.8__ is required for the AeroGear WebPush Server.
 
     cd server-netty
-    mvn exec:exec
+    mvn compile exec:exec
     (maven output not displayed)
     [main] INFO org.jboss.aerogear.webpush.netty.WebPushNettyServer - WebPush server bound to localhost:8443
-    
+
+### Start the Node.js server
+Follow the [Getting started instructions](https://github.com/kitcambridge/node-webpush-server#getting-started) to start
+the Node.js WebPush Server.
+
+    $ webpush-server --port 8080 --key demo.key --cert selfsigned.crt
+    Listening on https://Daniels-MBP.lan:8080...
+
+The _demo.key_ and _selfsigned.crt_ can be found in [server-netty/src/main/resources](../server-netty/src/main/resources).
+
 ### Start the console
-Press ```tab``` to see the available commands and also to display the available options for specific commands.   
+__Java 1.8__ is required for the WebPush Console when connecting to the AeroGear WebPush Server, and __Java 1.7__ for
+the node-webpush-server.
```

Comment:
@danbev How about:

**Java 1.8** is required for the WebPush Console when connecting to the _AeroGear WebPush Server_.
**Java 1.7** is required for the WebPush Console when connecting to the for the _node-webpush-server_.